test/test_flows.py

```diff
@@ -17,7 +17,7 @@
 
 def _make_iaf_args(input_dim, hidden_dims):
     _, rng_perm = random.split(random.PRNGKey(0))
-    perm = random.shuffle(rng_perm, onp.arange(input_dim))
+    perm = random.permutation(rng_perm, onp.arange(input_dim))
```

Comment:
The tests are still failing, I think there are a few more instances that we need to fix.

Reply:
Thanks Neeraj, it passed now.